types/docker-modem/index.d.ts

```diff
@@ -0,0 +1,119 @@
+// Type definitions for docker-modem 3.0
+// Project: https://github.com/apocas/docker-modem
+// Definitions by: Nasreddine Bac Ali <https://github.com/bacali95>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
```suggestion
```
it's already 3.4 (unless i"m wrong) which is minimal the best is to not specify version unless required by type features